pyconbalkan/timetable/migrations/0001_initial.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 2.0.5 on 2018-05-23 20:41
```

Comment:
@nadaj I just removed both migration files and created a initial migration since we didn't apply anything :)